app/uk/gov/hmrc/debttransformationstub/utils/ifsrulesmasterspreadsheet/impl/ConfigOrdering.scala

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.debttransformationstub.utils.ifsrulesmasterspreadsheet.impl
+
+object ConfigOrdering {
+
+  /** Avoids reordering existing config in the IFS `application.conf` file. */
+  object MainTransSubTransInIfsConfigOrdering extends Ordering[(String, String)] {
```

Comment:
This _could have been implicit_, but isn't on purpose. Ordering mainTrans and subTrans depends on the use case and I didn't want to accidentally reorder things I didn't intend to.